UWP/PPSSPP_UWPMain.cpp

```diff
@@ -367,12 +392,15 @@ std::vector<std::string> System_GetPropertyStringVec(SystemProperty prop) {
 		tempPath.resize(sz);
 		result.push_back(ConvertWStringToUTF8(tempPath));
 
-		if (getenv("TMPDIR") && strlen(getenv("TMPDIR")) != 0)
+		// As per below 'getenv' isn't supported
+		// https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/getenv-wgetenv?view=msvc-170
+		/*if (getenv("TMPDIR") && strlen(getenv("TMPDIR")) != 0)
```

Comment:
FWIW, this is used to tell a user there's a problem when their savedata is being stored in a temp folder (this happens if you launch the non-UWP build from within a zip file) since your savedata will get lost between runs.

We can just remove these lines for UWP and rely only on `GetTempPath()`.  It should be fine.

-[Unknown]